sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -27,7 +27,7 @@ Use FarmBeats client library for Python to do the following.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-verticals-agrifood-farming</artifactId>
-  <version>1.0.0-beta.2</version>
+  <version>1.1.0-beta.2</version>
```

Comment:
why update the version here? 

PS: refer [here](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#versions-and-versioning) for versioning.